src/current/v23.1/cpu-profiler.md

```diff
@@ -0,0 +1,61 @@
+---
+title: CPU Profiler
+summary: Verify that your cluster's data replication, data placement, and zone configurations are working as expected.
+keywords: cpu profile, cpu profiles, cpu profiler
+toc: true
+docs_area: manage
+---
+
+{% include_cached new-in.html version="v23.1" %} The CPU Profiler automatically captures CPU profiles. A CPU profile shows the functions that use the most CPU time, sampled over a window of time.
+
+Use CPU Profiles to troubleshoot spikes in CPU utilization or erratic CPU load on certain nodes. You can collect a CPU Profile manually on the [Advanced Debug page](ui-debug-pages.html). However, it may be difficult to manually capture a profile of a short CPU spike at the right point in time. Automatic CPU profile capture enables the investigation of CPU load in this and other cases, such as periodic high CPU.
+
+## Configuration
+
+You can configure automatic CPU profile capture with the following [cluster settings](cluster-settings.html):
+
+Cluster Setting | Description | Default Value | Recommended Value
+----------------|-------------|---------------|------------------
+`server.cpu_profile.enabled` | The ON/OFF switch of the CPU profiler | `false` | 
+`server.cpu_profile.cpu_usage_combined_threshold` | The baseline value for when CPU profiles should be taken. Collect profiles from each node that meets threshold. | MAX integer, such as `9223372036854775807` | `80`
+`server.cpu_profile.interval` | When the high water mark resets and a new CPU profile can be taken | `5m0s` (5 minutes) | `1m40s` (100 seconds)
```

Comment:
I think we should elaborate more here since this isn't the whole truth. While the high water mark does reset at this provided interval this may not be the only case when a profile can be taken. 

A profile is taken when the combined CPU percentage exceeds the current high water mark. When this happens, that value becomes the new high water mark. For e.g. if we start out with a default threshold is 80 and our combined CPU usage is 82 we take a profile and the high water mark is now 82. Next time around (in 1s), let's say the value is 85 then we will take a profile again and the high water mark now is 85. So we do take profiles in shorter time spans than the interval it just becomes more unlikely that a profile is taken since the value to exceed becomes higher. The high water mark will reset to 80 once the interval has passed.

Reply:
review above comment with @Santamaura

* so the cpu profiler (as well as the other automatic profilers) run asynchronously in the background and after every second we call the function which does all the logic to see whether we should take a profile or not
* so basically we check after every second whether we should take a profile or not
* we also don't take profiles if other profiles are already in progress

Another example:
if the `default server.cpu_profile.duration` is `10s` , the `server.cpu_profile.interval`  is `5m0s`, and `server.cpu_profile.cpu_usage_combined_threshold` is `80`:
- If at time0 the CPU exceeds the default threshold of 80 with 82 CPU, a 10s profile is taken and the high watermark becomes 82.
- After the 10s profile capture, the profiler continues to check every second if the CPU exceed 82.
- If at time1 the CPU is 85, another profile is taken for 10s and the high watermark becomes 85.
- At time2 (time0 + 5m0s ), the high watermark is reset to 80.

The diagram (taken from the enablement slide desk) is not exactly correct.

Reply:
added High-water mark section